server/api/email_handler.py

```diff
@@ -0,0 +1,31 @@
+import os
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+from flask import jsonify, request, Blueprint
+import json
+from config import SENDGRID_API_KEY
+
+email_handler = Blueprint('email_handler', __name__)
+
+
+@email_handler.route('/sendemail', methods=['POST'])
+def send_email():
```

Comment:
let's maybe make this a service (make it into a function, and put it in a folder called services, in a file called something like email_service.py or something) instead of a route so that we can call it from other routes!